cmake/FetchKLU.cmake

```diff
@@ -0,0 +1,29 @@
+# TODO: clean up this mess.
```

Comment:
In what sense? You should take care of this

Reply:
done, was a deprecated comment